examples/main_refactor.py

```diff
@@ -0,0 +1,165 @@
+"""
+Entry point for working with refactored code
+"""
+import os
+import warnings
+
+import numpy
+from scipy.io import savemat
+
+from tvb_epilepsy.base.constants import FOLDER_RES, FOLDER_FIGURES, SAVE_FLAG, SHOW_FLAG, SIMULATION_MODE, \
+    TVB, DATA_MODE
+from tvb_epilepsy.base.disease_hypothesis import DiseaseHypothesis
+from tvb_epilepsy.base.equilibrum_service import EquilibrumComputationService
+from tvb_epilepsy.base.plot_tools import plot_nullclines_eq, plot_sim_results
+from tvb_epilepsy.base.utils import initialize_logger, set_time_scales, calculate_projection, filter_data
+from tvb_epilepsy.custom.read_write import write_h5_model, write_ts_epi, write_ts_seeg_epi
+from tvb_epilepsy.tvb_api.epileptor_models import EpileptorDP
+
+if DATA_MODE is TVB:
```

Comment:
If `DATA_MODE` is a constant, why is it in a `if`-`else`? Better to use a command line flag for this.